source/_views/partials/tags.html

```diff
@@ -0,0 +1,7 @@
+<!-- array of post authors: this is where new authors are added.  Add the author into the set authors array as "first-last"-->
+{% set authors = ["nikki-stevens", "jason-savino", "mike-klass"] %} 
+<!-- conditional statement to skip author tags from appearing in posts and tags page -->
+{% if tag in authors %}
```

Comment:
I saw the docs had a "not" but they were within ifs too

On Tue, Aug 18, 2015 at 5:56 PM, Nikki Stevens <notifications@github.com>
wrote:

> In source/_views/partials/tags.html
> <https://github.com/waybetterdev/dev.waybetter.com/pull/10#discussion_r37370942>
> :
>
> > @@ -0,0 +1,7 @@
> > +<!-- array of post authors: this is where new authors are added.  Add the author into the set authors array as "first-last"-->
> > +{% set authors = ["nikki-stevens", "jason-savino", "mike-klass"] %}
> > +<!-- conditional statement to skip author tags from appearing in posts and tags page -->
> > +{% if tag in authors %}
>
> does twig support NOT in authors so you don't have to do an if and then
> also an else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/waybetterdev/dev.waybetter.com/pull/10/files#r37370942>
> .
>



-- 
Mahalo,


Teresa Nededog | Full Stack Software Engineer
804.252.7447 | Github <http://github.com/treindle> | Portfolio
<http://portfolio.reindle.io/>